lisp/mastodon-toot.el

```diff
@@ -69,10 +69,19 @@ Set `mastodon-toot--content-warning' to nil."
   (interactive)
   (mastodon-toot--kill))
 
+(defun mastodon-toot--remove-docs ()
+  "Get the body of a toot from the current compose buffer."
+  (let ((re "^|=+=|$"))
+    (save-excursion
+      (goto-char 0)
+      (re-search-forward re)
+      (re-search-forward re) ; end of the docs
```

Comment:
No need to call twice. The last argument on `re-search-forward` is count. 

So:

```elisp
(re-search-forward re nil nil 2)
```
is like calling it twice.